computestorage/setup.go

```diff
@@ -50,6 +50,9 @@ func SetupBaseOSLayer(ctx context.Context, layerPath string, vhdHandle windows.H
 // `volumePath` is the path to the volume to be used for setup.
 //
 // `options` are the options applied while processing the layer.
+//
+// NOTE: This API is only available on builds of Windows greater than 19645. The client
+// will need to handle if this is safe to call.
 func SetupBaseOSVolume(ctx context.Context, layerPath, volumePath string, options OsLayerOptions) (err error) {
 	if osversion.Build() < 19645 {
 		return errors.New("SetupBaseOSVolume is not present on builds older than 19645")
```

Comment:
Was the intent to remove this check as well?

Reply:
Oh geez, I never updated the comment after deciding which direction to take this PR. This check was there since the inception of this package actually, so was present in v0.9.x. I want to leave whatever was in a released tag as is, so I'm going to leave this here, and introduce a version where you can call it at your will and the client can handle figuring out whether it can safely call it. The comment needs to be swapped to 'you'll need to manifest to use this' or similar so thanks for catching this.